messaging/base/argo-workflows/send-message.yaml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
I think this can go to core actually as many workflows can use it? @goern @harshad16

Reply:
yeah, I agree, if this is not a standalone component. if this triggered by some other component then this should go in that application subdirectory.

Reply:
It can be used as workflow-helpers in different components basically :)

Reply:
Should I also add a sub-directory `argo-workflows` in `core/base`? To put the WorkflowTemplate in?

Reply:
yeah that would be appropriate space to add these